conan/tools/scm/git.py

```diff
@@ -76,12 +76,12 @@ def get_repo_root(self):
         folder = self._run("rev-parse --show-toplevel")
         return folder.replace("\\", "/")
 
-    def clone(self, url, target=""):
+    def clone(self, url, target="", args=[]):
```

Comment:
Recall that a list is not good for default args in python! :)
```suggestion
    def clone(self, url, target="", args=None):
        args = args or []
```